qa/tasks/cephfs/test_misc.py

```diff
@@ -223,6 +223,13 @@ def test_dump_inode_hexademical(self):
         info = self.fs.mds_asok(['dump', 'inode', hex(ino)])
         assert info['path'] == "/foo"
 
+    def test_dump_dir(self):
```

Comment:
minor nit: also add a test to check `dentry_dump` option.

Otherwise looks good.

Reply:
> minor nit: also add a test to check `dentry_dump` option.
> 
> Otherwise looks good.

OK! I have added it.